### ISSUE TITLE
Bug: Engine pointer events are not emitted

### ISSUE THREAD

**Issue by kamranayub** [excaliburjs/Excalibur#1439]
I was trying to subscribe to canvas pointer events via `engine.input.pointers` but it seems that they are no longer being emitted as I would expect

### Steps to Reproduce

```js
engine.input.pointers.on('up', () => {
  console.log('pointerup');
});
```

https://codesandbox.io/s/pedantic-ishizaka-pyjzn

### Expected Result

Console log should display when canvas is clicked anywhere

### Actual Result

No event is emitted

### Environment

- browsers and versions: Chrome 80
- operating system: Mac OS
- Excalibur versions: 0.23.0

### Current Workaround

It looks like this is fixed in `master` currently and should be released in 0.24.0

**Comment by kamranayub** [comment]
Quick update to this: it's still an issue in master but only reproducible when you try to attach an event handler during Scene.onInitialize

**Comment by eonarheim** [comment]
Good find, this feels like a order of operations issue in the initialize
lifecycle

**Comment by eonarheim** [comment]
It's possible this will be fixed by #1890 #1900
